### DISCUSSION TITLE
morethan-log using guide for Korean!

### DISCUSSION THREAD

**Discussion by morethanmin** [morethanmin/morethan-log#89] [Announcements]
This post was written by @jhk0530. 🙂 I think it would be okay to refer to the post until the Korean document is written.
Alternatively, if you ask in this thread or issue, I will try to answer as much as possible.

post: 
https://jhk0530.medium.com/notion-blog-with-morethanlog-3f4409e6f613

thanks!

**Comment by vigneshgarrapally** [comment]
Is there a similar blog in English?

**Reply by jhk0530** [reply]
maybe you can create new one

**Reply by vigneshgarrapally** [reply]
Only If I could figure out how to deploy

**Reply by jhk0530** [reply]
I'll consider writing a guide in English.
